src/components/Publish/_validation.ts

```diff
@@ -78,11 +80,23 @@ const validationPricing = {
       (param) => maxDecimalsValidation.test(param?.toString())
     )
     .required('Required'),
-  amountDataToken: Yup.number()
-    .min(50, (param) => `Must be more or equal to ${param.min}`)
-    .required('Required'),
+  amountDataToken: Yup.number().required('Required'),
   amountOcean: Yup.number()
-    .min(50, (param) => `Must be more or equal to ${param.min}`)
+    .test('validator-custom-name', '', function (value) {
```

Comment:
no clue what naming this method does but for the sake of completeness we should give it a fitting name, like `validator-min-amountOcean` or something